R/getAwudsData.R

```diff
@@ -28,6 +28,7 @@
 get_awuds_data <- function(awuds.data.path = NA, awuds.data.files = NA) {
 
   if ( !is.na(awuds.data.path) ) {
+    if ( !dir.exists(awuds.data.path) ) stop('Did not recieve a valid path.')
```

Comment:
This is a bit silly, but this will help users who pass a file in as the path input. (me!) Maybe we could just infer which is which and only have one input?